cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/polling/task/ClouderaManagerTemplateInstallationChecker.java

```diff
@@ -92,4 +62,21 @@ public String successMessage(ClouderaManagerCommandPollerObject clouderaManagerC
     protected String getCommandName() {
         return "Template install";
     }
+
+    private void fail(String messagePrefix, ApiCommand apiCommand, CommandsResourceApi commandsResourceApi) {
+        List<String> errorReasons = ClouderaManagerCommandApiErrorParserUtil.getErrors(apiCommand, commandsResourceApi);
+        String msg = messagePrefix + "Installation of CDP with Cloudera Manager has failed: [" + String.join(", ", errorReasons) + "]";
+        LOGGER.info(msg);
+        for (String errorReason : errorReasons) {
+            // Unfortunately CM is not giving back too much details about the errors, and what is returned usually nondeterministic:
+            // In a good case it returns "Failed to create HDFS directory",
+            // but sometines it just returns "Aborted command" or "Command timed-out after 186 seconds", so matching on such generic error error messages
+            // has no added value, therefore we are just checing whether AudiTDir related commands are failing or not.
```

Comment:
Typo: AudiTDir -> AuditDir. Also, please mention HMS WH & External WH dirs in the comment.

Reply:
Thanks. fixed.